[lldb] Temporarily disable actor state test

jasonmolenda opened the pull request:
The test added in "Add summary formatter for DefaultActorStorage (#10388)" is checking the actor's state, but it is failing sometimes. Temporarily disable until Dave can look into this, to unblock CI/PR testing.

rdar://148754221

jasonmolenda committed:
[lldb] Temporarily disable actor state test

The test added in "Add summary formatter for DefaultActorStorage (#10388)"
is checking the actor's state, but it is failing sometimes.
Temporarily disable until Dave can look into this, to unblock
CI/PR testing.

rdar://148754221

jasonmolenda commented:
@swift-ci test

felipepiovezan reviewed (approved):
Safest would be to just skip the test, but if this also temporarily gets the job done, lgtm

jasonmolenda commented:
Yeah if the test was flakey I wasn't sure if it's safe to run anywhere, so i'd be adding skipifDarwin, skipifWindows, skipifLinux.  maybe that is the better approach.

jasonmolenda committed:
Revert "[lldb] Temporarily disable actor state test"

This reverts commit 937c1859be608b8cdb4cfbee9140f47b45e01bd8.

jasonmolenda committed:
[lldb] Temporarily disable actor state test

The test added in "Add summary formatter for DefaultActorStorage (#10388)"
is checking the actor's state, but it is failing sometimes.
Temporarily disable until Dave can look into this, to unblock
CI/PR testing.

rdar://148754221

jasonmolenda commented:
@swift-ci test
